### PR TITLE
Readds clonexadone, nerfs cyroxadone's cloneloss healing

### PR DESCRIPTION
**Pull request by PKPenguin321**
:cl:
tweak: Cryoxadone's ability to heal cloneloss has been greatly reduced.
add: Clonexadone has been readded. It functions exactly like cryoxadone, but only heals cloneloss, and at a decent rate. Brew it with 1 part cryoxadone, 1 part sodium, and 5 units of plasma for a catalyst.
/:cl:

*Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:*

Two reasons!
1. Interdepartmental cooperation when it comes to cloning. Cloning lots of people in a decent amount of time used to require frequent trips/deliveries from chemistry to get more clonexadone. Now that you get two beakers of cryoxadone roundstart, it requires basically no effort at all to just toss a guy into the tube until it dings.
2. Creative cryo mixes. Nowadays I only ever really see mannitol and occasionally iron or mutadone mixed into the cryo beakers (seriously, this is such a big thing that [mannitol has become the second most brewed chem](https://atlantaned.space/newSS13tools/stats/viewMonthlyStats.php?year=2017&month=02)). Chemists will be greatly encouraged to modify the contents of the cryo mix with the addition to clonexadone, possibly inspiring them to experiment with other robust mixes again.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:-1:

The reason I removed clonexadone was because of the following:

1. There wasn't any "creative gameplay" with it. It was "stack clonex and cryox lol superfast omegaheal 420 no-MDscope", don't pretend cryomixes were anything more complex than this + maybe some tricord or something idk

2. >"Interdepartmental cooperation when it comes to cloning." isn't a thing when chemistry and medbay are the same department.

3. It was just the same fucking problem that the rest of the trek chems had, where the meds were just "tier 1->tier 2->tier 3" with no tradeoffs, no downsides, and nothing stopping you from just stacking them for max heals. It was boring, it was stupid, and it was bad design wise. Sure okay you got to heal really fast but it just meant that making anything but clonex+cryox isn't worth it. Speaking of which,

4. This buffs cryo a shitton. Buffing Cryo is bad. It's bad enough Medical Doctors are practically useless because you get killed for trying to stop assistants from doing your job for you, but buffing it even harder means I might as well just remove medical doctors because they just bodyblock the cure-all self service tubes you can have your ~~metagaming buddy~~ROLEPLAY FRIEND operate for you. If this gets merged, I'm removing Medical Doctor and making all the medbay doors leading to the cryotube lab all access to prevent ban requests over MDs being in the way from flooding in.

5. Chemists now have to produce the magic cure-all juice constantly again and will get whined at for making anything that isn't magic cure-all juice. Ever thought about not undoing all the work I did to **remove cure-alls in the first place?**

In short, this is just a cure-all cryo buff. I know you probably didn't intend it to be this way and you're probably just trying to do a good thing, but if you want medical to be medical and not the assistant self-service cureall tube room, don't do this.

Or just revert goofchem entirely and stop pussyfooting about it idk

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -156,6 +156,29 @@
...
+	name = "Clonexadone"
+	id = "clonexadone"
+	description = "A chemical that derives from Cryoxadone. It specializes in healing clone damage, but nothing else. Requires very cold temperatures to properly metabolize, and metabolizes quicker than cryoxadone."
+	color = "#0000C8"
+	taste_description = "muscle"
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/clonexadone/on_mob_life(mob/living/M)
+	switch(M.bodytemperature) // Low temperatures are required to take effect.
+		if(0 to 100) // At extreme temperatures (upgraded cryo) the effect is greatly increased.
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: oh wait so you didn't actually re-add clonex as it was originally you just made it handle specifically clone stuff?

this is much better but also just feels really needless

**Comment by ShadowDeath6** [comment]
this isn't stacking healchems though, if you add clonex to the mix it means less space for your cryox mannitol jerkoff combo whatever since it only heals clone damage

**Comment by Iamgoofball** [comment]
he didn't actually re-add clonex i got meme'd

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: this isn't how it always worked?

**Comment by PKPenguin321** [comment]
I *could* remove cryox's cloneloss healing entirely if you feel this makes cloning too easy, but then braindead MDs would shove dudes in the tubes who pop out of cloning and wonder why they never finish healing

**Comment by Iamgoofball** [comment]
It's not a matter of cloning being easy, it's a matter of MDs being relevant

**Comment by ShadowDeath6** [comment]
remove cryox's healing of everything else so you need to OD on bicaridine to heal in cryo

**Comment by PKPenguin321** [comment]
okay so I'm not sure if your gigapost up there is still relevant because you misinterpreted my PR but I'll try to respond anyways I guess?

1. all i know is i have fond memories of brainstorming the most efficient cryomix i could pack into 50u with reed glover in chemistry, and the experimentation aspect was engaging enough to take almost whole rounds for a least a few rounds. i miss that
2. that's kind of pedantic but i meant "having different parts of medbay work together to accomplish a common goal", like how viro can give genetics the defects virus, that kind of thing, you know?
3. not really relevant because this isn't what you had assumed it was, this clonex really isn't an upgrade to cryox at all
4. not really, more of a nerf, since cryo beakers dont start with clonex. if it gets clonex then it is a very slight (we're talking +1 per tick) buff to cloneloss heal rates
5. this one i really don't get, because **cryox is the magic cure-all chem!** nerfing cryox and having clonex as a separate thing makes it less of that. i'm not sure why cryox was buffed to be such a cure-all in the first place, seems like it does the opposite of the whole point of goofchem

the "mds being relevant" thing:
this makes them more relevant because:
THE GAME RIGHT NOW (no clonex)
- doctor gets man from cloner
- doctor puts man in automatically ejecting already cold already filled with cure-all cryox
- that's it

THE GAME WITH CLONEX
- doctor gets man from cloner
- doctor notices the clones are healing real slow
- doctor sees there is no clonex, the chemists are slackin off!
- doctor badgers the chemists to make some clonex you lazy bums/breaks in and makes some himself
- doctor puts the clonex into the cryo beaker and the patient now heals real fast and eventually pops out of the tube
- done

**Comment by Iamgoofball** [comment]
it was buffed because i didn't actually buff it i just merged cryox and clonex so people wouldn't whine so hard about cryo being nerfed

**Comment by Iamgoofball** [comment]
i hate cryox with a passion but we'd have riots if i removed it

**Comment by XDTM** [comment]
In any case, i think cloning goes to 100% now, so cloneloss is a speedcloning thing.

**Comment by PKPenguin321** [comment]
that's really dumb if true

**Comment by Ausops** [comment]
speedcloning is currently lethal no matter what mix you have, since you risk not having vital organs or limbs when popped out

**Comment by coiax** [comment]
@XDTM no, an unupgraded cloning pod will pop people out at around 40% health.

**Comment by ExcessiveUseOfCobblestone** [comment]
Eventually speedcloning is going to get remerged under the guise of a cryo chem that synthesize limbs for you in cold temperatures.

I honestly think this will be a gateway to that.

**Comment by Hatfish** [comment]
What if it's lowpop and there's no chemists around to make clonex?

**Comment by PKPenguin321** [comment]
Either slowclone or break in and make your own like I did back in the day

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (ToadStar) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Clonex used to heal better then Cryox in addition to the cloneloss healing.

**Comment by AdamElTablawy** [comment]
Competent chemists will still brainstorm 50u or 100u super-speed heal mixes in cryo, so this won't change much.

Either go the whole route and make medbay as a whole harder or don't fuck with it in the hopes that players will take the slightly more inconvenient route for greater efficiency, because nobody will care about anybody but themselves and some minor inefficiency isn't something they're going to bother with when there are valids to hunt.

**Comment by PKPenguin321** [comment]
*cough*

**Comment by jughu** [comment]
@hatfish implying 70% of the chemists arent making deathchems anyways
